src/applications/disability-benefits/all-claims/pages/homelessOrAtRisk.js

```diff
@@ -144,6 +144,12 @@ export const schema = {
         otherAtRiskHousing,
       },
     },
-    homelessnessContact,
+    homelessnessContact: {
+      ...homelessnessContact,
+      properties: {
+        ...homelessnessContact.properties,
+        name: _.omit('pattern', homelessnessContact.properties.name),
```

Comment:
Our future selves might appreciate a comment on why we're intentionally ignoring the regex in the schema.